common/common.h

```diff
@@ -58,11 +58,13 @@ typedef union {
 #include "math.h"
 #include "theme.h"
 #include "message-box.h"
+#include "power.h"
 
 void menuStartupPath(void);
 void menuStartup(void);
 void themeMenuStartup(void);
 void menuLoop(void);
+void menuExit(void);
```

Comment:
Doesn't exist anymore.

Reply:
Apologies, thought I removed that. Good catch.